### ISSUE TITLE
[Bug][PanelBar] aria-expanded is set to [Object object] when panel bar item is expanded

### ISSUE THREAD

**Issue by Yanko-Georgiev** [telerik/kendo-vue#730]
**Describe the bug**
The aria-expanded attribute is set to [Object object] when the panel bar item is expanded and custom content is set.

**To Reproduce**
 1. Open the following example: https://stackblitz.com/edit/cnkiksj9?file=src%2Fmain.vue
 2. Inspect the `My teammates` panel

**Expected behavior**
The aria-expanded attribute should be set to true.

Reported in TicketID: 1698736

**Comment by Yanko-Georgiev** [comment]
The fix is available in version 7.0.1-develop.1.
